index.html

```diff
@@ -1,4 +1,5 @@
 index.html
 Swithed to github-uncle9x9 which is fetched from origin
 Switched to newFeature1 branch and added line 3 into index.html
-This is a hotfix and apply to main directly.
```

Comment:
Please create branch to do the changes and please follow pull request to do at least one review and one approval.